pkgs/development/python-modules/pylibdmtx/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, pillow
+, numpy
+, libdmtx
+, pkg-config
+, lib
+}:
+
+buildPythonPackage rec {
```

Comment:
Could you add `pythonImportsCheck`?

Reply:
added it for grahamc but i don't personally use this package so i'll leave it to grahamc to merge it